frontend/freebase2wikidata.js

```diff
@@ -260,8 +260,85 @@ $(function() {
     var title = mw.config.get('wgTitle');
     return qidRegEx.test(title) ? title : false;
   }
-  
-  (function generateNav() { 
+
+    ps.openNav = function openNav(itemLabel, propertyLabel, propertyValue, referenceURL, buttons) {
+        $('#myNav').width('100%');
+
+        var blackboard = $('#blackboard');
+
+        blackboard.append(buttons.eq(0).clone(true, true));
+        blackboard.append(buttons.eq(1).clone(true, true));
+
+        blackboard.append('<div class="loader"></div>');
+
+        $.ajax({
+            type: 'GET',
+            url: 'https://tools.wmflabs.org/strephit/search?url=' + encodeURI(referenceURL),
+            success: function(msg) {
+                $('.loader').remove();
+
+                if(msg != 'no preview'){
+                    var data = JSON.parse(msg);
+                    var regEx = new RegExp('(' + itemLabel + '|' + propertyLabel + '|' + propertyValue + ')(?!([^<]+)?>)', 'gi');
+
+                    if(!data.hasOwnProperty('excerpt'))
+                    {
+                        blackboard.append('<h1><a href="' + data.url + '">' + data.url + '</a></h1>');
+                        blackboard.append('<ul></ul>');
+                        
+                        var linklist = $('#blackboard > ul');
+                        for(var index in data){
+                            var item = data[index];
+                            if(index != 'url' && item !== null){
+                                if( index == 'other' ) {
+                                    while(typeof item == 'string'){
+                                        item = JSON.parse(item);
+                                    }
+                                    for(var key in item){
+                                        if(item[key] !== null){
+                                            linklist.append(('<li>' + '<strong>' + key + '</strong>' + ': ' + item[key] + '</li>').replace(regEx, '<span class="highlight">$1</span>'));
+                                        }
+                                    }
+                                }
+                                else {
```

Comment:
This file seems to use the K&R coding style. Example:
```
if(foo) {
   bar;
} else {
   baz;
}
```

It's not important but it's better for consistency.

Reply:
Will fix this